internal/local/app_test.go

```diff
@@ -83,6 +83,36 @@ func TestLoadApp(t *testing.T) {
 	assert.Equal(t, expectedAppLocal, app)
 }
 
+func TestLoadConfig(t *testing.T) {
+	wd, wdErr := os.Getwd()
+	assert.Nil(t, wdErr)
+
+	for _, config := range []struct {
+		version realm.AppConfigVersion
+		file    File
+	}{
+		{realm.AppConfigVersion20180301, FileStitch},
+		{realm.AppConfigVersion20200603, FileConfig},
+		{realm.AppConfigVersion20210101, FileRealmConfig},
+	} {
+		t.Run(fmt.Sprintf("should successfully load config file with version %d", config.version), func(t *testing.T) {
+			projectRoot := filepath.Join(wd, "testdata", config.version.String(), "local")
+			testApp := App{RootDir: projectRoot, Config: config.file}
+
+			err := testApp.LoadConfig()
+			assert.Nil(t, err)
```

Comment:
[style/meganit] keeps a tighter scope if you inline these types of assertions
```suggestion
			assert.Nil(t, testApp.LoadConfig())
```

it honestly _shouldnt_ matter when we write perfect code...

but ive been burned enough times (read: once) by the following:

1) re-assigning to an `err` variable in a test
2) failing to assert on it
3) the compiler never notices the "un-used" var (b/c it was re-assigned with `=`, not newly assigned with `:=`)
4) i'm left wondering why the heck my test isn't catching the error I expect to be returned 😬 

that's why i now tend to push for the tightly scoped vars whenever possible